Remotion/Web/Development.WebTesting.IntegrationTests/ChromeBinariesProviderTest.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;

```

Comment:
Missing license header

Reply:
Fixed in https://github.com/re-motion/Trunk-SVN2Git/pull/12/commits/cc21cb58b42c8d277dc5010ce41de8afefffb1e3.